.github/workflows/release.yml

```diff
@@ -10,13 +10,22 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
```

Comment:
setup-go has a v2 release, don't know if it is worth upgrading: https://github.com/actions/setup-go#v2